Header header
string label
sensor_msgs/PointCloud2 cloud
uint64 timestamp
geometry_msgs/TransformStamped transformToGlobal
geometry_msgs/Vector3 sweepCenter
geometry_msgs/Vector3 objectPositionGlobal
#For using the mongodb geospatial search option
std_msgs/Float32MultiArray loc
string waypoint
